pyproject.toml

```diff
@@ -0,0 +1,22 @@
+[tool.poetry]
```

Comment:
Is Poetry needed here if pip is used for the lambda?

I think we need to use one or the other here not both as the repo only has the lambda.

Reply:
Moved to PIP